tests/test_app.py

```diff
@@ -1049,6 +1049,201 @@ async def test_bulk_specification_success():
+            #     }
+            # }
+            assert resp.status == 400
+
+
+async def test_bulk_specification_dts_fail_json_without_dts():
+    async with AppClient(config) as cli:
+        with FileUtil() as fu:
+            sub_dir = "dts_folder"
+            dts_dir = f"testuser/{sub_dir}"
+            fu.make_dir(dts_dir)  # testuser is hardcoded in the auth mock
+            base = Path(fu.base_dir) / "testuser" / sub_dir
+            manifest_1 = "test_manifest_1.json"
+            manifest_1_dict = {
+                "resources": [],
+                "instructions": {
+                    "protocol": "KBase narrative import",
+                    "objects": [
+                        {
+                            "data_type": "gff_metagenome",
+                            "parameters": {
+                                "fasta_file": "fasta_1",
+                                "gff_file": "gff_1",
+                                "genome_name": "mg_1",
+                            },
+                        }
+                    ],
+                },
+            }
+            with open(base / manifest_1, "w", encoding="utf-8") as f:
+                json.dump(manifest_1_dict, f)
+            resp = await cli.get(f"bulk_specification/?files={sub_dir}/{manifest_1}")
+            jsn = await resp.json()
+            assert jsn == {
+                "errors": [
+                    {
+                        "type": "cannot_parse_file",
+                        "file": f"{dts_dir}/{manifest_1}",
+                        "message": "json is not a supported file type for import specifications",
+                        "tab": None,
+                    }
+                ]
+            }
+            assert resp.status == 400
+
+
+async def test_bulk_specification_dts_fail_wrong_format():
+    async with AppClient(config) as cli:
+        with FileUtil() as fu:
+            sub_dir = "dts_folder"
+            dts_dir = f"testuser/{sub_dir}"
+            fu.make_dir(dts_dir)  # testuser is hardcoded in the auth mock
+            base = Path(fu.base_dir) / "testuser" / sub_dir
+            manifest = "test_manifest.json"
+            manifest_data = ["wrong", "format"]
+            with open(base / manifest, "w", encoding="utf-8") as f:
+                json.dump(manifest_data, f)
+            resp = await cli.get(f"bulk_specification/?files={sub_dir}/{manifest}&dts")
+            jsn = await resp.json()
+            assert jsn == {
+                "errors": [
+                    {
+                        "type": "cannot_parse_file",
+                        "file": f"{dts_dir}/{manifest}",
+                        "message": "Manifest is not a dictionary",
+                        "tab": None,
+                    }
+                ]
+            }
+            assert resp.status == 400
+
+
+async def test_bulk_specification_dts_fail_wrong_extension():
```

Comment:
Should also add a test for a file without an extension

Reply:
Done.